pages/read.tsx

```diff
@@ -13,10 +13,15 @@ export const getServerSideProps = async (context) => {
   const { url } = context.query;
   const htmlContent = await fetchHtml(url);
   const doc = new JSDOM(htmlContent, { url });
+  const DOMPurify = createDOMPurify(doc.window);
   const reader = new Readability(doc.window.document);
   const article = reader.parse();
   return {
-    props: { article }
+    props: {
+      article: Object.assign({}, article, {
```

Comment:
I'm curious why don't we just use object spread in this case?

Reply:
I'm just a horse 🐴 Sorry for that, will change to object spread as you want.